solutions/standard/DA-types.md

```diff
@@ -0,0 +1,121 @@
+# IBM Cloud Elasticsearch Deployable Architecture Variables Documentation
+This document provides detailed information about the complex object types used in the IBM Cloud Elasticsearch DA (Deployable Architecture).
+
+
+## 1. Service Credential Names
+
+**Description:**
+This variable defines a map of service credential names and their corresponding roles for the Elasticsearch database. This allows for the creation of service credentials with specific roles, providing controlled access to the database.
+
+**Type:**
+The `service_credential_names` variable is a map where the key is the name of the service credential and the value is the role assigned to that credential.
+
+**Default Value:**
+The default value for the `service_credential_names` variable is an empty map (`{}`).
+
+### Detailed Structure:
+
+- `Key (name, required):` The name of the service credential.
+
+- `Value (role, required):` The role assigned to the service credential.
```

Comment:
suggest to give an example of what a typical value could look like

Reply:
Done.

Reply:
Provided example.